roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -129,6 +129,8 @@
   include_role:
     name: network_plugin/calico
     tasks_from: check
+  vars:
```

Comment:
If you do this, you are essentially disabling the CNI each time since it will override any value passed from ansible group vars. Why do you need to do this?